audit.json

```diff
@@ -94,6 +96,7 @@
   "10096_Timestamp Disclosure - Unix_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/callback/add-hearing-bookings/about-to-start_POST": "ignore",
   "10096_Timestamp Disclosure - Unix_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/notify-gatekeeper/about-to-start_POST": "ignore",
   "90033_Loosely Scoped Cookie_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/case-submission/mid-event_POST": "ignore",
+  "90033_Loosely Scoped Cookie_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/remove-order/about-to-start_POST": "ignore",
```

Comment:
Do you wanna add cookie stuff for the new mid event and about to submit as well?

Reply:
Just wondering whether we should try and be proactive or reactionary with them

Reply:
Good point, I was thinking the same as well. I will add.

Reply:
I think we should be running script just to add everything :P